### PR TITLE
Fix checking whether the ranking is enabled on team scoreboard.

### PR DESCRIPTION
**Pull request by meisterT**

### CODE REVIEW

**Review by nickygerritsen — APPROVED** [review]

File: webapp/src/Controller/Jury/ConfigController.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ public function indexAction(EventLogService $eventLogService, Request $request):
                 }
             }
             $before = $this->config->all();
-            $errors = $this->config->saveChanges($data, $eventLogService, $this->dj, $options);
+            $errors = $this->config->saveChanges($data, $eventLogService, $this->dj, true, $options);
```
Comment (vmcj) [inline webapp/src/Controller/Jury/ConfigController.php:69]: Can this be a named property?

File: webapp/tests/Unit/BaseTestCase.php [hunk 2]
```diff
@@ -322,6 +322,6 @@ protected function setupDataSource(int $dataSource): void
         $config   = self::getContainer()->get(ConfigurationService::class);
         $eventLog = self::getContainer()->get(EventLogService::class);
         $dj       = self::getContainer()->get(DOMJudgeService::class);
-        $config->saveChanges(['data_source'=>$dataSource], $eventLog, $dj);
+        $config->saveChanges(['data_source'=>$dataSource], $eventLog, $dj, false);
```
Comment (vmcj) [inline webapp/tests/Unit/BaseTestCase.php:325]: Same here.

**Review by vmcj — APPROVED** [review]

File: webapp/src/Controller/Jury/ConfigController.php (hunk 1 above)
Comment (meisterT) [reply webapp/src/Controller/Jury/ConfigController.php]: Done.

File: webapp/tests/Unit/BaseTestCase.php (hunk 2 above)
Comment (meisterT) [reply webapp/tests/Unit/BaseTestCase.php]: Done.

**Pull request merged**